eye2bids/edf2bids.py

```diff
@@ -520,8 +525,11 @@ def edf2bids(
 
     events_json.input_file = input_file
     events_json.two_eyes = _2eyesmode(df_ms_reduced)
+    events_json.TaskName = _has_TaskName(events)
+
+    if events_json.TaskName:
```

Comment:
I think we can use a wallruss operator here.

Will explain when amin front of a proper computer.